src/pageobjects/workbench/Input.ts

```diff
@@ -47,8 +47,10 @@ export abstract class Input extends BasePage<AllInputLocators> {
         const currentText = await this.getText()
         if (currentText !== text) {
             await clipboard.write(text)
-            const backSpaces = new Array(currentText.length).fill('Backspace')
-            await input.addValue(backSpaces)
+            if (currentText?.length) {
```

Comment:
My assumption here is that if there is no text we don't have to enter any backspaces but happy to be corrected.